src/storyblocks.js

```diff
@@ -1,37 +1,49 @@
-const createError = require('http-errors');
+const _createError = require('http-errors');
+const camelCase = require('lodash.camelcase');
 const crypto = require('crypto');
 const got = require('got');
+const isPlainObject = require('lodash.isplainobject');
 const mapKeys = require('lodash.mapkeys');
 const snakeCase = require('lodash.snakecase');
 
 const AUTH_EXPIRY_SECONDS = 12 * 60 * 60; // 12 hours
 
-// strip any \u0000 null characters from the response before it is json parsed.
+function createError (code, errorOrErrors) {
+  if (typeof errorOrErrors === 'string') return _createError(code, errorOrErrors);
+  const errors = mapKeys(errorOrErrors, (value, key) => camelCase(key));
```

Comment:
```suggestion
  const errors = Object.keys().map(key => camelCase(key));
```
Would this not give the same result (and be a little more vanilla)?

Reply:
@timlesallen not really...
```js
const o = { test_value: 42 };
mapKeys(o, (value, key) => camelCase(key)); // = { testValue: 42 }
Object.keys().map(key => camelCase(key)); // = ['testValue']
```